apps/builder/next.config.mjs

```diff
@@ -81,8 +81,17 @@ const nextConfig = {
               (process.env.NEXT_PUBLIC_POSTHOG_HOST ??
                 'https://app.posthog.com') + '/:path*',
           },
+          {
+            source: '/healthz',
```

Comment:
Why `healthz`?

Reply:
In Kubernetes, they use "/healthz" as the default path for health checks (liveness and readiness probes). That's why I used the "/healthz" route. I can change it to the "/health" route if you prefer.